schema/api.py

```
@@ -9,13 +9,22 @@
 from pymongo import ASCENDING, DESCENDING
 from datetime import datetime, timedelta
 import math
+import os
 from schema.serializers import serialize_vulnerability, serialize_all_vulnerability, nvd_serializer, mitre_serializer, serialize_githubpocs
 from gevent import spawn, joinall
```

Ruff (0.8.2)

`gevent.spawn` imported but unused

Remove unused import: `gevent.spawn`

(F401)